test/utils/react.js

```diff
@@ -30,3 +30,13 @@ export function mockComponent(sinon, module, mockTagName = 'div', props = null)
         );
     });
 }
+
+export function getContext(context = {}) {
+    const defaultContext = {
+        app: {},
+        settings: {},
+        ui: {}
+    };
+
+    return Object.assign(defaultContext, context);
```

Comment:
I prefer by far the object spread syntax over object.assign. However, Object spread is still a proposal and it's available because we use a babel transform.